### PR TITLE
Lint fixes for exercises: complex-numbers, etl, gigasecond

### PR DESCRIPTION
**Pull request by trvrfrd**
For issue #480 

Bundled up a few simple ESLint fixes in one convenient PR.

complex-numbers:
```
javascript/exercises/complex-numbers/complex-numbers.spec.js
  45:1  error  Trailing spaces not allowed  no-trailing-spaces

javascript/exercises/complex-numbers/example.js
   1:36  error  Block must not be padded by blank lines  padded-blocks
  19:58  error  Expected newline before ')'              function-paren-newline
  27:64  error  Expected newline before ')'              function-paren-newline
  41:48  error  Expected newline before ')'              function-paren-newline
```

etl:
(I actually don't love these changes, but they make ESLint happy)
```
javascript/exercises/etl/etl.spec.js
  13:22  error  Expected a line break after this opening brace   object-curly-newline
  13:53  error  Expected a line break before this closing brace  object-curly-newline
  20:22  error  Expected a line break after this opening brace   object-curly-newline
  20:47  error  Expected a line break before this closing brace  object-curly-newline
```

gigasecond:
```
javascript/exercises/gigasecond/example.js
  3:27  error  Expected parentheses around arrow function argument having a body with curly braces  arrow-parens
  6:2   error  Missing semicolon                                                                    semi
```

Happy to squash this into one commit if deemed appropriate.

Thank you for your time!

### CODE REVIEW

File: exercises/gigasecond/example.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const GIGASECOND_IN_MILIS = 1e9 * 1e3;
```
Comment (matthewmorgan) [inline exercises/gigasecond/example.js]: I hate to ask, but while we're here, can we fix the spelling of 'millis'?  Should be `MILLIS`... :)

**Comment by matthewmorgan** [comment]
@trvrfrd looks awesome, thanks for helping us clean this up.  One request, if you don't mind...

File: exercises/gigasecond/example.js (hunk 1 above)
Comment (trvrfrd) [reply exercises/gigasecond/example.js]: Good eye! I did not even catch that. Will fix.

**Comment by matthewmorgan** [comment]
Thanks @trvrfrd !!

**Pull request merged**